packages/renderer/src/lib/onboarding/OnboardingEmbeddedComponent.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+import { onMount } from 'svelte';
+import { configurationProperties } from '/@/stores/configurationProperties';
+import { providerInfos } from '/@/stores/providers';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../main/src/plugin/configuration-registry';
+import type { ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import PreferencesConnectionCreationRendering from '../preferences/PreferencesConnectionCreationRendering.svelte';
+import type { OnboardingEmbeddedComponentId } from '../../../../main/src/plugin/api/onboarding';
+
+export let embeddedComponent: OnboardingEmbeddedComponentId;
+export let extensionId: string;
+
+let configurationItems: IConfigurationPropertyRecordedSchema[];
+$: configurationItems;
+let providers: ProviderInfo[];
+let providerInfo: ProviderInfo | undefined;
+$: providerInfo;
+
+onMount(() => {
+  configurationProperties.subscribe(value => {
+    configurationItems = value;
+  });
+
+  providerInfos.subscribe(value => {
+    providers = value;
+    providerInfo = providers.find(p => p.extensionId === extensionId);
+  });
+});
+</script>
+
+{#if embeddedComponent === 'create' && providerInfo && configurationItems}
+  {#if providerInfo?.containerProviderConnectionCreation === true}
+    <PreferencesConnectionCreationRendering
+      providerInfo="{providerInfo}"
+      properties="{configurationItems}"
+      propertyScope="ContainerProviderConnectionFactory"
+      callback="{window.createContainerProviderConnection}"
+      isEmbedded="{true}" />
+  {/if}
+
+  <!-- Create connection panel-->
+  {#if providerInfo?.kubernetesProviderConnectionCreation === true}
+    <PreferencesConnectionCreationRendering
+      providerInfo="{providerInfo}"
+      properties="{configurationItems}"
+      propertyScope="KubernetesProviderConnectionFactory"
+      callback="{window.createKubernetesProviderConnection}"
+      isEmbedded="{true}" />
+  {/if}
+{/if}
```

Comment:
what happens if a user say I want to include the wizard of creating a podman machine and the provider is not providing the feature ?

I guess we should display something like 'this provider does not support this mode'